pheme/templatetags/charts/pie.py

```diff
@@ -120,9 +126,16 @@ def pie_chart(
             font_size=font_size,
         ).strip()
 
-    legend = _build_legend(chart_size + 10, chart_size, title_color)
+    legend_start = calculate_legend_start_height(
+        chart_size, title_color, font_size
+    )
+    legend = build_legend(legend_start, title_color)
     return SafeText(
         __PIE_CHART_TEMPLATE.format(
-            size=chart_size, height=chart_size + 26, donut=donut, legend=legend
+            start_pie=width - chart_size,
+            size=chart_size + (width - chart_size),
```

Comment:
I guess this is only `width` ...?

Reply:
No, the first one start position of the pie chart and the second is the overall size of that circle.

Reply:
Nevermind, yes it is.